### DISCUSSION TITLE
AlDente 1.08

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#417] [General]
* Updated Libraries to comply with macOS 12
* Fixed an issue that caused textfields not to work on macOS 12
* Improved license verification for pro users
* Improved clamshell discharge for pro users
* Sailing Mode now persists unplugging of the power cable
* Fixed a bug that occured after helper removal
* Added German Language support
* Many more improvements & fixes

[Enjoy your Pasta,](https://www.youtube.com/watch?v=QoeUbnpv4xI)
David & Matthias

<hr /><em>This discussion was created from the release <a href='https://github.com/davidwernhart/AlDente/releases/tag/'>AlDente 1.08</a>.</em>

**Comment by ange335** [comment]
I bought the pro version and installed it (link in email) , immediately there was an update notification. AlDente 1.08 free version was installed as an update and there was only the "go pro" option which leads to the website, no possibility to enter the license again.

I uninstalled it an reinstalled the pro version from the link in the email and now I got in a installer loop. I have to confirm the installation of the helper, confirm that I should disable Mac OS battery control and than the loop starts. The software asked again to install the helper... Look at the screenshot, when I click install helper thong happened now.

Device: MacBookPro 18,3 (M1 Pro 14 inch)
OS: macOS 12.1 (21C5021h)

<img width="469" alt="Screen 2021-10-31 um 16 29 23" src="https://user-images.githubusercontent.com/78279658/139590781-114db397-3666-4ca5-9ede-fa7db8e2984a.png">

**Reply by MatthiasKerbl** [reply]
Hi @ange335,

Sorry for the inconvenience, by now, all links should be updated to the newest version.

If you still experience this issue, you can fix it like that:

1. Quit/Delete/Uninstall everything from AlDente.
2. Download the latest version directly here from Github: https://github.com/davidwernhart/AlDente/releases/download/1.08/AlDente1.08.dmg
3. Installation should work without issues now

Cheers,
Matthias
